src/components/SearchResultTile/styledComponents.tsx

```diff
@@ -0,0 +1,83 @@
+import styled from "styled-components";
+
+import { minDevice } from "../../assets/styles/breakpoints";
+import { textDetails, white, lightGrey } from "../../assets/styles/colors";
+import P from "../P/styledComponents";
+import Wrapper from "../Wrapper/styledComponents";
+
+const SearchResultTileWrapper = styled(Wrapper)`
+  box-shadow: 0px 2px 8px rgba(0, 0, 0, 0.1);
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  background-color: ${white};
+  margin: 0.3125em 0;
+  padding: 0.5em 1em;
+  border: 0.5px solid ${lightGrey};
+  box-sizing: border-box;
+  border-radius: 5px;
```

Comment:
Just a nitpick, you can try to group similar properties together, here border-related, you can have similar groups with fonts and spacings etc.